datafusion/core/tests/fuzz_cases/aggregate_fuzz.rs

```diff
@@ -36,6 +36,7 @@ use datafusion_physical_expr::{AggregateExpr, PhysicalSortExpr};
 use test_utils::add_empty_batches;
 
 #[cfg(test)]
+#[allow(clippy::items_after_test_module)]
```

Comment:
TBH I don't understand why these are wrapped into a tests module at all, since this IS already an integration test file (lives under `tests`). So I think clippy is not wrong here: this is confusing.

Could you try to just erase the additional `mod tests` indirection?

Reply:
I tried this suggestion and it works well. I will make a follow on PR to further simplify the code

Reply:
https://github.com/apache/arrow-datafusion/pull/8036